src/dotnet-install.ps1

```diff
@@ -176,6 +176,16 @@ function Measure-Action($name, $block) {
     Say-Verbose "⏱ Action '$name' took $totalSeconds seconds"
 }
 
+function Get-File-Size($zipUri) {
+    $response = Invoke-WebRequest -Uri $zipUri -Method Head
+    $fileSize = [long]$response.Headers["Content-Length"]
+    $fileSizeBits = $fileSize * 8
```

Comment:
Why are the sizes converted to bits?

Reply:
@donJoseLuis did you have any specific reason for requesting this?